website/addons/wiki/views.py

```diff
@@ -422,6 +428,12 @@ def project_wiki_rename(auth, wname, **kwargs):
     else:
         sharejs_uuid = wiki_utils.get_sharejs_uuid(node, new_wiki_name)
         wiki_utils.broadcast_to_sharejs('redirect', sharejs_uuid, node, new_wiki_name)
+    if auth.user.comments_viewed_timestamp.get(node._id, None) and \
```

Comment:
This can probably be simplified by breaking this into smaller pieces, something like:

```python
comments_timestamp = auth.user.comments_viewed_timestamp.get(node._id, {})
wiki_comments_timestamp = comments_timestamp.get('wiki', {})
wiki_page_timestamp = wiki_comments_timestamp.get(wiki_name, None)
if  wiki_page_timestamp:
    # ...
```